### ISSUE TITLE
About stride support?

### ISSUE THREAD

**Issue by clclon** [stbrumme/toojpeg#5]
Do you have any plans to add stride support for the input buffer?

**Comment by stbrumme** [comment]
No, but I'm working on a substantial rewrite of toojpeg which supports more input formats, such as RGBA, BGR, YUV, etc.
I will take a look at the performance impact ("runtime cost") and decide whether it's worth it or not.

**Comment by clclon** [comment]
Thanks in advance for possible support in this matter.  there is not much work, you need to take into account the difference between width and stride, skipping this difference for processing.

For example: the real image has dimensions 600x1024, the stride is 608. You need to process as usual 600 pixels horizontally, and skip 8 pixels by rewinding the buffer.

If necessary, I can lay out a buffer with a reference stride as a .raw file
